sdk/servicebus/Azure.Messaging.ServiceBus/Directory.Build.props

```
@@ -9,9 +9,8 @@
     These analyzers are blocked by https://github.com/Azure/azure-sdk-tools/issues/127
      -->
     <NoWarn>
-      $(NoWarn);
-      AZC0006;
       AZC0007;
+      $(NoWarn);

```

```suggestion
      $(NoWarn);
       AZC0007;
```

By convention, we start with the defined set and extend.